src/components/FollowersCard.jsx

```diff
@@ -59,7 +59,7 @@ const FollowersCard = ({ follower }) => {
           <p>
             {`${followerProfile.followers} Followers, ${followerProfile.following} Following`}
           </p>
-          <p>{`Public Repository : ${followerProfile.public_repos}`}</p>
+          <p>{`${followerProfile.public_repos} Public Repositories`}</p>
```

Comment:
We could add some lines which can change the word Repositories to Repository if user has only a single repository.

Reply:
👍